src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -484,6 +498,23 @@ export class ChallengesubmitComponent implements OnInit {
   }
 
   validateInput(inputValue) {
-    this.inputFile = inputValue === null;
+    const regex = new RegExp(/https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/);
+    const validExtensions = ['json', 'zip', 'csv'];
+    if (this.onSubmissionOption = false) {
```

Comment:
I think this variable needs updation here.

Reply:
the if condition above wasn't working but i corrected it but what updation it requires